pages/02.documentation/16.pod-security-policy/default.en.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Pod Security Policy'
+taxonomy:
+    tag:
+        - metakube
+        - security
+        - psp
+        - podsecuritypolicy
+---
+
+By default Pod Security Policies are deactivated in MetaKube clusters, but you can optionally activate them on the cluster detail page in the dashboard. Pod Security Policies allow you to specify fine-grained authorization settings for Pod creation and updates.
+
+! If you activate Pod Security Policy, a lot of Pod specifications, Operators and Helm charts will not work out of the box anymore. Only activate it if you know what you are doing.
+
+## Configuring Pod Security Policy
+
+You can activate Pod Security Polices on the cluster details page under "Configure Pod Security Policy".
+
+![Configure Pod Security Policy](psp.png)
+
+## How does it work
+
+The Kubernetes documentation has a detailed article about [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/). You should read this before activating the feature.
+
+### Example
+
+After activating Pod Security Policies, every Pod needs a Policy in order to be started or updated.
+
+If you create a new deployment:
+
+```bash
+cat <<'EOF' | kubectl apply --namespace default -f -
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: nginx-hello
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx-hello
+  template:
+    metadata:
+      labels:
+        app: nginx-hello
+    spec:
+      containers:
+      - name: nginx-hello
+        image: nginxdemos/hello:0.2
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
+          timeoutSeconds: 1
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 80
+          timeoutSeconds: 1
+EOF
+```
+
+You can see that the Deployment and a subsequent ReplicaSet were created, but the pods can not be scheduled:
```

Comment:
Reading it again, I would replace "but the pods can not be scheduled" with "but the pods were not" because the pods aren't even created (i.e. the scheduler isn't involved).